src/OpenTelemetry.Extensions.Hosting/OpenTelemetryMetricsBuilderExtensions.cs

```diff
@@ -0,0 +1,70 @@
+// <copyright file="OpenTelemetryMetricsBuilderExtensions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using OpenTelemetry.Internal;
+using OpenTelemetry.Metrics;
+
+namespace Microsoft.Extensions.Diagnostics.Metrics;
+
+/// <summary>
+/// Contains extension methods for registering OpenTelemetry metrics with an
+/// <see cref="IMetricsBuilder"/> instance.
+/// </summary>
+internal static class OpenTelemetryMetricsBuilderExtensions
+{
+    /// <summary>
+    /// Adds an OpenTelemetry <see cref="IMetricsListener"/> named 'OpenTelemetry' to the <see cref="IMetricsBuilder"/>.
+    /// </summary>
+    /// <remarks>
+    /// Note: This is safe to be called multiple times and by library authors.
```

Comment:
nit: We could remove this for now as it's an internal method.

Reply:
I think it is still good documentation for the intent/behavior. I'm leaning towards leaving it in there.